infra/modules/azure_app_service/outputs.tf

```
@@ -16,10 +16,16 @@ output "app_service" {
       id           = azurerm_linux_web_app.this.id
       name         = azurerm_linux_web_app.this.name
       principal_id = azurerm_linux_web_app.this.identity[0].principal_id
+      pep = {
+        record_sets = azurerm_private_endpoint.app_service_sites.private_dns_zone_configs.record_sets
+      }
```

if there are nothing else from _pep_ that we need, we can do this
```suggestion
      pep_record_sets = azurerm_private_endpoint.app_service_sites.private_dns_zone_configs.record_sets
```